src/06-sort.js

```diff
@@ -23,7 +26,16 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+  return songs.sort((a, b) => {
```

Comment:
Make sure to include the possibility of a === b so you don't leave room for any error